src/js/utils/string-utils.js

```diff
@@ -16,37 +16,36 @@
          * indicated via curly braces, e.g. 'Hello {name}'. These tokens are
          * replaced by the parameter value of the same name.
          *
-         * Parameter values will be rendered using their toString methods and then
-         * HTML-escaped. The only exception is that instances of the class HTML
-         * are rendered without escaping as their contract declares that they are
-         * already valid HTML.
+         * Parameter values will be rendered using their toString methods.
+         * **NO** HTML escaping or sanitizing of any form is performed.
+         * If HTML escaping is required (for example, if user supplied input is
+         * being interpolated), use HtmlUtils.interpolateHtml().
          *
          * Example:
          *
          *~~~ javascript
-         * HtmlUtils.interpolate(
-         *     'You are enrolling in {spanStart}{courseName}{spanEnd}',
+         * let comments = ['hello', ':)'];
+         * StringUtils.interpolate(
+         *     '{n} comments',
          *     {
-         *         courseName: 'Rock & Roll 101',
-         *         spanStart: HtmlUtils.HTML('<span class="course-title">'),
-         *         spanEnd: HtmlUtils.HTML('</span>')
+         *         n: comments.length,
          *     }
          * );
          *~~~
          *
          * returns:
          *
          *~~~ javascript
-         * 'You are enrolling in <span class="course-title">Rock &amp; Roll 101</span>'
+         * '2 comments'
```

Comment:
I think that `'You are enrolling in {courseName}'`, where `courseName: 'Rock & Roll 101',`, would be a good example that shows that `&` is not escaped in the output.